apps/hpc-ftsadmin/src/app/utils/parse-filters.ts

```diff
@@ -84,12 +93,12 @@ const filterValueIsFormObjectValue = (
 };
 
 const filterValueIsArrayFormObjectValue = (
-  value: FilterValues
+  value: FilterValue
 ): value is util.FormObjectValue[] => {
   return Array.isArray(value) && typeof value[0] !== 'string';
 };
```

Comment:
Simiar comments here
```suggestion
const filterValueIsFormObjectValueArray = (
  value: FilterValue
): value is util.FormObjectValue[] => {
  return Array.isArray(value) && value.every(filterValueIsFormObjectValue);
};
```